src/SEAL/README.md

```diff
@@ -22,6 +22,24 @@ $document = [
     'id' => '1',
     'title' => 'New Blog',
     'article' => '<article><h2>Some Subtitle</h2><p>A html field with some content</p></article>',
+    'blocks' => [
+        [
+            'type' => 'text',
+            'title' => 'Titel',
+            'description' => '<p>Description</p>',
+            'media' => ['id' => 1, 'displayOption' => 'top'],
```

Comment:
The tricky part is supporting this kind of things here. A property which need to be handled differently based on the type.